resources/styles/book-content/tips-for-success.less

```diff
@@ -16,25 +18,43 @@
     padding: 0 10px;
     margin: 0;
   }
-  > ul:first-child {
-    padding-top: 10px;
-    border-top: 1px solid @tutor-neutral-light;
+  p {
+    #fonts > .sans(1.7rem, 1.5em);
+    color: @tutor-neutral-dark;
+    padding: 15px 20px;
+    border-left: 1px solid @tutor-neutral-lighter;
+    border-right: 1px solid @tutor-neutral-lighter;
+    background: @tutor-neutral-lightest;
+    &:first-child {
```

Comment:
That way the first <p> has a top border and the last one has a bottom border. I would prefer to place one border on the containing element, but that wasn't possible.